woocommerce/woocommerce-admin
# Onboarding: Add toggles to configured payments in task list

Fixes (part of) #3710

*.Adds payment toggles to already completed payment methods.
* Checks payment configuration client-side
* Moves payment methods to their own file.
* Fxes an error with checking requested settings and errors in wc-api.

### Screenshots
<img width="706" alt="Screen Shot 2020-03-02 at 5 25 00 PM" src="https://user-images.githubusercontent.com/10561050/75696002-2d749700-5cab-11ea-9879-63d728e42910.png">


### Detailed test instructions:

1. Delete the option `woocommerce_task_list_payments` if you need to reset.
1. Configure at least one payment gateway.
1. Make sure the payment gateway's toggle is initially checked.
1. Try toggling this payment gateway on/off and make sure the setting is reflected in `wp-admin/admin.php?page=wc-settings&tab=checkout`
